ZipBuilder/Sources/ZipBuilder/ResourcesManager.swift

```diff
@@ -80,28 +161,22 @@ extension ResourcesManager {
   ///                  directory if it doesn't exist.
   /// - Returns: An array of URLs pointing to the newly located bundles.
   /// - Throws: Any file system errors that occur.
+  @discardableResult
```

Comment:
I learned something!

Reply:
:D it's very nice, especially for testing (which would be nice to add eventually too...) - use the result in the testing but if you don't need it at the call site it's handy 🎉